### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by skkr**
This includes the final button styles that match the size of input elements, and adjustments on badges, plus adjustments for related styles.

Note that the titles indicate the usage purpose, for implementation we should use the indicated class names as before.

Modifiers not inherited from bootstrap, follow the BEM naming convention. This also makes easier to identify them.

Also includes the assets from wt-ui-assets and doc updates on the branding section.

image

Buttons can include icons

image

### CODE REVIEW

File: src/css/settings/_bootstrap-variables.scss [hunk 1]
```diff
@@ -210,9 +210,10 @@ $grid-gutter-width:           30px;
 // $border-width:                1px;
 // $border-color:                $gray-300;
 
-$border-radius:               4px;
-// $border-radius-lg:            .3rem;
-// $border-radius-sm:            .2rem;
+$border-radius:               4px; //4px
+$border-radius-lg:            4px; //.3rem
+$border-radius-sm:            4px; //.2rem
```
Comment (PabloAlice) [inline src/css/settings/_bootstrap-variables.scss]: why are they equal?

File: src/stories/feedback.stories.js [hunk 2]
```diff
@@ -26,40 +26,29 @@ storiesOf('Feedback', module)
...
-
-      <button type="button" className="btn btn-primary">
-        Notifications <span className="badge badge-light">4</span>
-      </button>
-
+      <p className="p-1">
+        <span className="mr-1 badge badge-primary">Primary</span>
+        <span className="mr-1 badge badge-secondary">Secondary</span>
+        <span className="mr-1 badge badge--accent">Accent</span>
+        <span className="mr-1 badge badge-success">Success</span>
```
Comment (PabloAlice) [inline src/stories/feedback.stories.js]: will we have success, danger, warning... styles?

File: src/css/settings/_bootstrap-variables.scss (hunk 1 above)
Comment (skkr) [reply src/css/settings/_bootstrap-variables.scss]: Is a design decision they look the same regardless the viewpoint size

File: src/stories/feedback.stories.js (hunk 2 above)
Comment (skkr) [reply src/stories/feedback.stories.js]: That's right

**Pull request merged**